script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_3.txt

```diff
@@ -65,15 +65,15 @@ What can I do for you, Shiki-sama?"
 <Page19>
 "Ah, nothing―――"
 It's hard to say 'I just wanted to thank you for making my bed.'
-I just need to find some other topic...ah, that's right.
+I just need to find some other topic... ah, that's right.
 <Page20>
-"I was wondering how Akiha was doing. Her school is a boarding school, right?"
+"I was wondering how Akiha is doing. Her school is a boarding school, right?"
 <Page21>
 "It is, indeed.
 It is true that the school Akiha-sama attends is a boarding school. However, as an exception, she is allowed to go to school from home."
 "...In other words, #you mean she goes to school from this house?"
 <Page22>
-"Yes, but she rarely comes home in the evening like today. After a full day of schoolwork, she has lessons to attend, so she usually returns shortly before seven o'clock."	
+"Yes, but she rarely comes home in the evening like today. After a full day of schoolwork, she has lessons to attend, so she usually returns shortly before seven o'clock."
```

Comment:
> はい。ですが、今日のように夕方に帰られる事は稀です。一日の学業がお済みになった後は習い事がおありなので、お帰りになられるのは決まって７時前となります

This is a little ambiguous as is since it sounds like she returns _to school_ shortly before seven o'clock which I don't think it says. Maybe I'm misunderstanding the Japanese, but it should either be:
> Yes, but she rarely comes home in the evening like to day. After a school day of school work, she has lessons to attend. She usually comes home shortly before seven o'clock.

though 7 is well under the umbrella of 夕方 so I'm not confident in what she's saying. Either way this should be cleared up (and split into two sentences).

Reply:
Since there's a られる on returning in the evening I might tweak that to "but she is rarely able to make it home before evening like today"? Otherwise agree with the intent though